YakimS/assignment1/Q1

```diff
@@ -0,0 +1,71 @@
+public class Assignment1Q1
+{
+
+    public static void main(String[] args)
```

Comment:
Ideally, to make the class fully reusable and clean (by clean I mean not containing unnecessary code) this class should be split into two - one that contains main logic and separate one used to test it, in this case containing `main` method.